src/app/containers/manuscript/index.tsx

```diff
@@ -9,15 +9,15 @@ import { ManuscriptEditor } from './manuscript-editor';
 
 const renderBackdrop = () => (
   <Backdrop open={true}>
-    <CircularProgress color='inherit' />
+    <CircularProgress color="inherit" />
```

Comment:
Looking at the eslintrc file, prettier is configured for single quotes. Does this show as an error if you run the linter?